examples/xgboost_ray_nyctaxi.py

```diff
@@ -5,8 +5,9 @@
 # Please refer to https://docs.ray.io/en/latest/xgboost-ray.html to install it.
 from xgboost_ray import RayDMatrix, train, RayParams
 import raydp
+from raydp.spark.dataset import RayMLDataset
```

Comment:
Why need import RayMLDataset twice? Can this line be removed?

Reply:
Removed